jsf/subsystem/src/main/java/org/jboss/as/jsf/subsystem/JSFExtension.java

```diff
@@ -69,13 +70,19 @@ public void initialize(final ExtensionContext context) {
         JSFLogger.ROOT_LOGGER.debug("Activating JSF(Mojarra) Extension");
         final SubsystemRegistration subsystem = context.registerSubsystem(SUBSYSTEM_NAME, CURRENT_MODEL_VERSION);
         subsystem.registerSubsystemModel(JSFResourceDefinition.INSTANCE);
-        subsystem.registerXMLElementWriter(JSFSubsystemParser_1_0.INSTANCE);
+        subsystem.registerXMLElementWriter(JSFSubsystemParser_1_1.INSTANCE);
+
+        // Transformers
+        if (context.isRegisterTransformers()) {
+            JSFResourceDefinition.registerTransformers(subsystem);
```

Comment:
There's a new ServiceLoader based way of registering transformers that should be used -- see implementations of org.jboss.as.controller.transform.ExtensionTransformerRegistration